### PR TITLE
Fix combobox variables conflicts in basic examples page

### PR DESCRIPTION
**Pull request by limonte**
We could test GitHub CR process on simple PRs as this one :)

Fix `combobox` variables conflicts by renaming them to `comboboxN`

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-combo-box/346)

### CODE REVIEW

File: demo/index.html [hunk 1]
```diff
@@ -48,8 +48,8 @@
           var combobox = document.querySelector('#demo2');
           combobox.items = elements;
 
-          combobox.addEventListener('value-changed', function() {
-            document.querySelector('#selected-value').innerHTML = combobox.value;
+          combobox.addEventListener('value-changed', function(e) {
+            document.querySelector('#selected-value').innerHTML = e.target.value;
```
Comment (platosha) [inline demo/index.html]: IMO, `combobox.value` is easier to get then `e.target.value`. I like that it explicitly tells that the value property is in the combobox.

Can we fix this without changing the example code this way?

Comment (samiheikki) [inline demo/index.html]: One example: https://github.com/vaadin/vaadin-combo-box/commit/264b4ac9a77e36db025f8b415ef534b33b30a1ce#diff-25dd8f443c3ef3c623850d3a204e916c

Comment (limonte) [inline demo/index.html]: > IMO, combobox.value is easier to get then e.target.value

Without doubts, it is easier, but I don't have a better solution here except adding `var combobox = e.target;` before that line which using `combobox.value`. Thoughts?

> One example: 264b4ac#diff-25dd8f443c3ef3c623850d3a204e916c

Well, adding numbers to variables isn't a good way of naming. Example: I'm adding the new example between 2dn and 3rd examples. If I follow the `comboboxN` naming convention, then I will need to rename all examples from 3rd to the last. Manually :)

Comment (limonte) [inline demo/index.html]: `comboboxN` is the lesser evil :) Done.

**Comment by platosha** [comment]
LGTM 👍

**Comment by manolo** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/346#-:-KX6VrKk7373AnngABX4:bnfp4nl)*

**Comment by manolo** [comment]
@samiheikki & @platosha could you ACK ?

**Comment by samiheikki** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/vaadin/vaadin-combo-box/346#-:-KX9hE53eiojzlZM9jdk:bnfp4nl)*

**Pull request merged**